### DISCUSSION TITLE
Add more donation options, add cryptocurrency.

### DISCUSSION THREAD

**Discussion by canisllup** [transmission/transmission#4140] [Support] [answered]
Transmission uses bittorrent - A decentralized file sharing protocol, but only accepts donations from centralized sources. IMO it should add options for accepting cryptocurrency.

**Comment by ckerr** [comment, marked as answer]
I don't want to accept BC donos. I like the decentralization but am not a fan of BC's carbon footprint.

If Transmission were making a lot of money I might feel differently, but it's not and so I feel like I can vote my conscience here. :smile_cat:

**Reply by canisllup** [reply]
Thanks for the timely response. I agree with you on btc, while there are alternatives which are both decentralized and eco-friendly, I will refrain from mentioning them since your point about revenue from donations was made clear. transmission-daemon is the least demanding torrent daemon on my ARMV5 NAS, thanks.

**Comment by robd003** [comment]
I'd like to suggest donations via Ethereum and Cardano, both of which don't rely on Proof of Work so they are more efficient than Bitcoin.

Ethereum moved from Proof of Work to Proof of Stake and dropped energy usage by 99.95%

Cardano has always used Proof of Stake and is even more energy efficient than Ethereum.

**Comment by nixorokish** [comment]
seconding Ethereum donations. Would donate with ETH! Tiny carbon footprint

**Comment by kaloudis** [comment]
@ckerr check out accepting Bitcoin payments via Lightning. Your carbon emission concern is mitigated as these transactions aren't recorded on the blockchain. hope you consider!

**Comment by redelvis** [comment]
@ckerr Hey, I love your work with Transmission. It seems like right now is the only option to donate is by using Paypal account, even if I want to use my credit card.

 Would be great to add an option to pay with durectly with CC (Stripe maybe?) without actually using Paypal which is not available in some countries.
